### PR TITLE
update README

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,37 +108,40 @@ Generate Terraform variables:
    seed_vm_flavor = "general.v1.small"
    seed_disk_size = 100
 
-   multinode_flavor     = "changeme"
-   multinode_image      = "CentOS-stream8-lvm"
+   multinode_flavor     = "general.v1.medium"
+   multinode_image      = "overcloud-rocky-9-yoga-20231026T081556"
```
Comment (MoteHue) [inline README.rst:112]: Is there any particular reason to use this image, and not ``Rocky9-lvm``?

Comment (Alex-Welsh) [reply README.rst:112]: `Rocky9-lvm` gets updated _somewhat_ regularly. Pinning to a specific image improves reliability

Comment (Alex-Welsh) [reply README.rst:112]: Plus making it more accurate to production is better

Comment (Alex-Welsh) [reply README.rst]: Changed my mind, I've just remembered that the overcloud* images have some awkward issues with vault at the moment that haven't yet been resolved

Comment (MoteHue) [reply README.rst]: > Changed my mind, I've just remembered that the overcloud* images have some awkward issues with vault at the moment that haven't yet been resolved

Will want to change the Ubuntu image mentioned later in the document too

Comment (Alex-Welsh) [reply README.rst]: Done

**Review by markgoddard — APPROVED** [review]

**Pull request merged**